package.json

```diff
@@ -162,6 +162,7 @@
     "marky": "^1.2.1",
     "mime-types": "^2.1.24",
     "mongodb-extjson": "^4.0.0-rc1",
+    "mongodb-schema": "^8.2.5",
```

Comment:
Since `compass-schema` also uses the `mongodb-schema` package https://github.com/mongodb-js/compass-schema/blob/master/package.json#L165 do we want the `mongodb-schema` dependency to be in `peerDependencies` and `devDependencies` so that Compass sets the version and we avoid possible duplicates from differing versions? Looks like the dependency isn't updated much though.